src/tpm2_wrap.c

```diff
@@ -2524,6 +2570,7 @@ int wolfTPM2_NVCreateAuth(WOLFTPM2_DEV* dev, WOLFTPM2_HANDLE* parent,
 {
     int rc;
     NV_DefineSpace_In in;
+    TPM2B_NAME name;
```

Comment:
You can use `nv->handle.name` directly. A separate TPM2B_NAME is not required.

Reply:
Indeed. I was already doing it in WriteAuth and ReadAuth. Good catch. Thank you